web/src/icons/Delete.jsx

```diff
@@ -0,0 +1,13 @@
+import { h } from 'preact';
+import { memo } from 'preact/compat';
+
+export function Delete({ className = '' }) {
+  return (
+    <svg className={`fill-current ${className}`} viewBox="0 0 24 24">
+      <path d="M0 0h24v24H0V0z" fill="none" />
+      <path d="M6 21h12V7H6v14zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z" />
+    </svg>
+  );
+}
+
+export default memo(Delete);
```

Comment:
we dont need to memo here

Reply:
This is a common pattern for icons, since they don't typically take any other arguments

Reply:
Just keeping this consistent with the other icons.